app/src/main/java/id/rizmaulana/covid19/data/model/CovidDetail.kt

```diff
@@ -9,15 +9,14 @@ import kotlinx.android.parcel.Parcelize
  * rizmaulana 04/03/20.
  */
 
-@Parcelize
-data class CovidDetail(
+@Parcelize data class CovidDetail(
     @Expose @SerializedName("confirmed") val confirmed: Int = 0,
-    @Expose @SerializedName("countryRegion") val countryRegion: String? = null,
+    @Expose @SerializedName("countryRegion") val countryRegion: String = "",
```

Comment:
Refer to https://github.com/rizmaulana/kotlin-mvvm-covid19/pull/2#discussion_r390191156